### ISSUE TITLE
Invalid schema (containing field "example_type")

### ISSUE THREAD

**Issue by hrennau** [APIs-guru/openapi-directory#788]
In the document `squareup.com.json` downloaded today (2021-03-09), an invalid named schema was found, containing a field `example_type`.

Location: `/components/schemas/CatalogObject/example_type`

**Comment by MikeRalphson** [comment]
Now has an unresolvable internal `$ref`, which is odd...

```
Cannot resolve reference: #/components/schemas/invoice_delivery_methodInvoiceDeliveryMethod
#/components/schemas/Invoice/properties/delivery_method/%24ref
```

**Comment by MikeRalphson** [comment]
Resolved by the fixes in #785 - thanks!
